src/prelude.rs

```diff
@@ -35,7 +35,7 @@ pub use crate::server;
 ///////////////////////////////////////////////////////////////////
 /// Structs
 ///////////////////////////////////////////////////////////////////
-pub use crate::frame::{Request, Response};
+pub use crate::frame::{Request, Response, SlaveRequest};
```

Comment:
My remarks also apply to `SlaveRequest` that should neither be part of the `prelude` nor visible without the "server" feature.

Reply:
if I don't put it in prelude it won't compile because the frame mod is not publicly exported in `lib.rs`
I don't know what would be the best way to handle this for this project. Do you have any recommendation ?

Reply:
Good point. Then keep it in the prelude and hide it behind the "server" feature.